docs/tests/getting-started/azure-pipelines.yml

```diff
@@ -0,0 +1,26 @@
+trigger:
+  branches:
+    include:
+    # Trigger this pipeline only on commits pushed to the main branch
+    - 'main'
+
+pool:
+  # Specifying which AzureDevOps hosted build pool to use
+  vmImage: 'ubuntu-latest'
+
+variables:
+  # Specifying which version of dagger to install
+  # This value is read from the install script
+  DAGGER_VERSION: 0.2.19
+
+jobs:
+- job: dagger
+  steps:
+  - script: |
+      cd /usr/local
+      curl -L https://dl.dagger.io/dagger/install.sh | DAGGER_VERSION=$(DAGGER_VERSION) sh
+    displayName: Install Dagger $(DAGGER_VERSION)
```

Comment:
You put quotes in `azure-pipelines-windows` but not here

Reply:
Removed quotes to keep the tasks uniform